src/megatron/bridge/training/utils/flop_utils.py

```
@@ -183,6 +187,40 @@ def transformer_flops():
+
+        if is_lora:
+            _LORA_SEQ_STATS = {
+                4096: (842603, 4096),
+                2048: (488991, 2030),
+            }
+            seq_len = cfg.model.seq_length
+            if seq_len not in _LORA_SEQ_STATS:
+                raise ValueError(
+                    f"No LoRA sequence statistics available for seq_length={seq_len}. "
+                    f"Add an entry to _LORA_SEQ_STATS for this sequence length."
+                )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)